experimental/dds/tree2/src/feature-libraries/typed-schema/flexList.ts

```diff
@@ -3,39 +3,79 @@
  * Licensed under the MIT License.
  */
 
+/** @remarks Used by `markEager`and `isMarkedEager`. */
+const flexListEager = Symbol("FlexList Eager");
+
+/** Mark the given object as an "eager" item in a `FlexList`, regardless of its type. */
+export function markEager<T>(t: T): T {
```

Comment:
This doesn't define eager.

Also it should prabably state that it returns its input.

Also its input should be constrained to object.

Maybe this should be renamed markEagerIfNeeded or something like that because it does not always make the change.

Reply:
Maybe note that this is only needed on functions, which is why this is used instead of just including the symbol when declaring the object?

Also when we move to class based schema building, we will prabably just put a single static property on the a base class to handle this instead of using this function.